cypress.config.ts

```diff
@@ -6,5 +6,7 @@ export default defineConfig({
       // implement node event listeners here
     },
     baseUrl: "http://localhost:3000",
+    video: false,
+    screenshotOnRunFailure: false,
```

Comment:
Maybe it would be nice to have screenshots on failure? I think keeping them ignored from the repository is okay for this.